invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/Stats/components/MostLoanedDocumentsList/MostLoanedDocumentsList.js

```diff
@@ -64,97 +63,105 @@ export default class MostLoanedDocumentsList extends Component {
 
   renderDateRangePicker() {
     return (
-      <Segment.Group>
-        <Segment>
-          <Header
-            as="h3"
-            content="Pick range of dates"
-            subheader="Choose between which dates to search the most loaned documents"
-          />
-        </Segment>
-        <Segment.Group horizontal>
-          <Segment>
-            <DateInput
-              autoComplete="off"
-              clearable
-              clearIcon={<Icon name="remove" color="red" />}
-              closable
-              dateFormat="YYYY-MM-DD"
-              iconPosition="left"
-              maxDate={this.state.toDate}
-              name="fromDate"
-              onChange={this.handleDateChange}
-              placeholder="Start Date"
-              value={this.state.fromDate}
-            />
-          </Segment>
-          <Segment>
-            <DateInput
-              autoComplete="off"
-              clearable
-              clearIcon={<Icon name="remove" color="red" />}
-              closable
-              dateFormat="YYYY-MM-DD"
-              iconPosition="left"
-              minDate={this.state.fromDate}
-              name="toDate"
-              onChange={this.handleDateChange}
-              placeholder="End Date"
-              value={this.state.toDate}
-            />
-          </Segment>
-        </Segment.Group>
-      </Segment.Group>
-    );
-  }
+      <Container>
+        <Header as="h3">
+          Pick range of dates
+          <Header.Subheader>
+            Choose between which dates to search the most loaned documents
+          </Header.Subheader>
+        </Header>
+
+        <List horizontal>
+          <List.Item>
+            <List.Content>
+              <DateInput
```

Comment:
There is a `<DatePicker` component that you might want to use to have a global coherent look and feel.

Reply:
So in the end no re-use of the DatePicker?